Doc/tools/extensions/glossary_search.py

```diff
@@ -15,6 +15,8 @@
 
 
 logger = logging.getLogger(__name__)
+STATIC_DIR = '_static'
+JSON = 'glossary.json'
```

Comment:
Minor suggestion: `JSON` -> `GLOSSARY_JSON`

Reply:
Well, the extension is called "glossary_search", in the context of the extension, I don't think that it's worth it to repeat "glossary" ;-)